system/Build.jl

```diff
@@ -2,7 +2,16 @@ if VERSION < v"1.9.0-DEV"
   error("Julia >= 1.9 required")
 end
 
+oscarpath = pkgdir(Oscar)
+
+println("=====================")
+println(oscarpath)
+println("=====================")
+
 using Pkg
+Pkg.activate(temp=true)
+Pkg.add(path="$(oscarpath)")
```

Comment:
I think we want 
```suggestion
Pkg.develop(path="$(oscarpath)")
```
as I think `Pkg.add` will copy stuff. Indeed:
```
julia> Pkg.add(path="/Users/mhorn/Projekte/OSCAR/AbstractAlgebra.jl")
    Updating git-repo `/Users/mhorn/Projekte/OSCAR/AbstractAlgebra.jl`
   Resolving package versions...
    Updating `/private/var/folders/d_/1zss2fnd6xdgclqnj8jj_27m0000gp/T/jl_N9fIWI/Project.toml`
  [c3fe647b] + AbstractAlgebra v0.31.1 `/Users/mhorn/Projekte/OSCAR/AbstractAlgebra.jl#master`
    Updating `/private/var/folders/d_/1zss2fnd6xdgclqnj8jj_27m0000gp/T/jl_N9fIWI/Manifest.toml`
...
julia> using AbstractAlgebra

julia> pkgdir(AbstractAlgebra)
"/Users/mhorn/.julia/packages/AbstractAlgebra/zuBl1"
```

In contrast:
```
julia> Pkg.develop(path="/Users/mhorn/Projekte/OSCAR/AbstractAlgebra.jl")
   Resolving package versions...
    Updating `/private/var/folders/d_/1zss2fnd6xdgclqnj8jj_27m0000gp/T/jl_ezgpmP/Project.toml`
  [c3fe647b] + AbstractAlgebra v0.31.1 `~/Projekte/OSCAR/AbstractAlgebra.jl`
    Updating `/private/var/folders/d_/1zss2fnd6xdgclqnj8jj_27m0000gp/T/jl_ezgpmP/Manifest.toml`
...

julia> using AbstractAlgebra

julia> pkgdir(AbstractAlgebra)
"/Users/mhorn/Projekte/OSCAR/AbstractAlgebra.jl"
```

Reply:
This seems to have fixed it